UIKIT_DZ_Clone_AppStore/UIKIT_DZ_Clone_AppStore/ViewControllers/GoodsForYouViewController/GoodsForYouViewController.swift

```diff
@@ -9,4 +9,198 @@ import UIKit
 
 final class GoodsForYouViewController: UIViewController {
 
+    // MARK: - Visual Components
+    private let scrollView = UIScrollView()
+    private let Barbutton  = UIButton()
```

Comment:
```suggestion
    private let barbutton  = UIButton()
```
ой а что у нас тут такое не 10 отжиманий случайно?)

Reply:
да блин...